packages/dds/tree/src/test/feature-libraries/modular-schema/comparison.spec.ts

```diff
@@ -165,11 +175,37 @@ describe("Schema Comparison", () => {
 				storedEmptyFieldSchema,
 				fieldRequiredEmptyTree,
 				fieldRequiredEmptyTree,
+				sequenceField,
+				identifierField,
```

Comment:
I'm surprised by this: doesn't `sequenceField` allow a superset of documents compared to `identifierField`?

Reply:
Ah, thank you. Would be nice if the function took a set instead of an array, or if the function were named `testPartialOrderable`. Not necessarily worth delaying this PR though.